.github/ISSUE_TEMPLATE/install_request.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Install request
+about: Request an installation of package in the stack on HPC's
+title: '[INSTALL] <title>'
+labels: 'install'
+assignees: 'Hang-Lei-NOAA, kgerheiser, jkbk2004'
```

Comment:
Needs updating

Reply:
Jong Kim is from the EPIC team, he is doing the installs of hpc-stack on Cheyenne as far as I know.